gems/sorbet/test/snapshot/partial/db_schema/expected/err.log

```diff
@@ -0,0 +1 @@
+Cloning into '<cache>/sorbet/sorbet-typed'...
```

Comment:
You could also omit this file if you wanted.

Reply:
Good point, I've removed it now.